addendas/mabe.go

```diff
@@ -0,0 +1,295 @@
+package addendas
+
+import (
+	"encoding/xml"
+	"errors"
+	"fmt"
+
+	"github.com/invopop/gobl.cfdi/internal"
+	"github.com/invopop/gobl.cfdi/internal/format"
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/org"
+)
+
+// Mabe schema constants
+const (
+	MabeVersion        = "1.0"
+	MabeNamespace      = "https://recepcionfe.mabempresa.com/cfd/addenda/v1"
+	MabeSchemaLocation = "https://recepcionfe.mabempresa.com/cfd/addenda/v1/mabev1.xsd"
+	MabeNotApplicable  = "NA"
+)
+
+// TipoDocumento valid values
+const (
+	MabeTipoDocumentoFactura     = "FACTURA"
+	MabeTipoDocumentoNotaCredito = "NOTA CREDITO"
+	MabeTipoDocumentoNotaCargo   = "NOTA CARGO"
+)
+
+// Mabe specific identity codes.
+const (
+	MabeIdentityTypeCode        = "MABE"
+	MabeRef1IdentityTypeCode    = "MABE-REF1"
+	MabeRef2IdentityTypeCode    = "MABE-REF2"
+	MabePlantIDIdentityTypeCode = "MABE-PLANT-ID"
+)
```

Comment:
@cavalle I'm not totally happy with the naming here... it feels inconsistent with the extensions. The current version of GOBL org.Identity however doesn't have support for using keys after the move to extensions...

Reply:
It might be worth re-instating this if the migration has been completed correctly: https://github.com/invopop/gobl/blob/main/org/identity.go#L29-L32